### ISSUE TITLE
[Tarea 3] Pregunta 3 - Formato grupo.txt

### ISSUE THREAD

**Issue by VicenteMerino** [UC-IIC3253/2021#53]
Hola, da lo mismo el formato en que guardemos p, g y q en grupo.txt? En particular podemos guardarlo como json?

**Comment by VicenteMerino** [comment]
Misma pregunta para como guardar en private_key.txt y public_key.txt, lo podemos definir nosotros?

**Comment by marceloarenassaavedra** [comment]
Aunque hoy respondimos esto en clases, me parece bueno que la respuesta también quede en el foro. Ustedes pueden decidir el formato que van a usar para almacenar `p`, `g`, `q` y las llaves pública y privada. En particular, en el caso de la llave pública no es necesario almacenar los valores `p`, `g` y `q` ya que los pueden acceder desde el archivo `grupo.txt`, y lo mismo ocurre para la llave privada.

Saludos!

**Comment by Okhan97** [comment]
@marceloarenassaavedra ¿donde especificamos el formato de grupo.txt?
¿Creo un README dentro de la carpeta 3? ¿Lo dejo como comentario al principio del archivo de la pregunta2?

**Comment by marceloarenassaavedra** [comment]
Sí, pueden especificar el formato de grupo.txt con un README dentro de la carpeta 3.

Saludos!
